content/docs/use-cases/fast-data-storage-layer.md

```diff
@@ -0,0 +1,143 @@
+# Fast Data Storage Layer
+
+Datasets used in data science tend to exceed typical storage and networking
```

Comment:
you mean "stretch the limits of" rather than "exceed?"

Reply:
I was thinking straight up exceeding but do you think stretch the limits of reads better? I'm fine with either one.

Reply:
well technically if it "exceeds" it wouldn't be "used" since it would be "unusable" right?

Reply:
Not necessarily, it just implies data science usually can't happen on typical computing systems.

Reply:
"tend to exceed" ≠ exceed. This is good, but "stretch the limits" may be better.

Reply:
I think that "stretch the limits" implies it never actually exceeds though, no? But that's an important scenario here actually, e.g. you simply don't have enough space in your local machine.

Reply:
I like when it turns into a semantics discussion :) Both of them are fine with me 👍🏼

Reply:
Haha yeah minor thing. I like this way better but no strong opinion, of you both vote for "stretch" I'll change it.